_guides/themes.md

```diff
@@ -33,21 +33,20 @@ TypeDoc ships with two themes so you can start creating docs right out of the bo
 
 ## Custom themes
 
-Creating a custom theme in TypeDoc is pretty easy. Basically you create a directory and overwrite the
-components of the [``default`` theme](https://github.com/TypeStrong/typedoc-default-themes/tree/master/src/default) 
-you want to change. All themes extend the default theme, so you only have to change the parts that don't 
-fit your needs.
+Creating a custom theme in TypeDoc is pretty easy. Basically you copy the  [default theme directory](https://github.com/TypeStrong/typedoc-default-themes/tree/master/src/default) (from which all themes extend from) and update the parts you want to change.
 
-This is the basic directory structure of a theme, only the assets directory is mandatory:
+Then pass a path to the theme to typedoc using the `--theme` option
 
+```bash
+$ typedoc --out path/to/documentation/ --theme path/to/custom-theme path/to/typescript/project/
 ```
-+ my-theme             // Root directory of your theme
-  + assets             // Static assets like *.css or *.js files
-  + layouts            // Optional. Layout templates
-    - default.hbs      // Optional. Default layout template
-  + partials           // Optional. Partials to be used by Handlebar
-  + templates          // Optional. Page templates
-    - index.hbs        // Optional. Home page template
-    - reflection.hbs   // Optional. Definition page template
-  - theme.js           // Optional. JavaScript class of the theme
-```
+
+The styles included by typedoc are then static CSS files generated by the typedoc-default-themes build, 
+not the original SASS files. You may use whatever process you prefer to generate the CSS files you want to include.
+
+This means if you want to add your own custom styles you have two options: override styles *or* compile your own sass.
+
+### Overriding styles
+
+1. In your custom directory add your custom css to `/assets/css/override.css`
+2. In your custom theme directory under `/layouts/default.hbs` add `<link rel="stylesheet" href="{{relativeURL "assets/css/override.css"}}">`
```

Comment:
@aciccarello  I pulled your other changes and now I can't find the old comments. I think you wanted me to remove everything from lines 46 down so that we would keep it open to them implementing however they want. Is that right?